reference/constraints/Choice.rst

```diff
@@ -367,10 +367,13 @@ too many options per the `max`_ option.
 strict
 ~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
+
+The validator will also check the type of the input value.
+
+.. caution::
 
-If true, the validator will also check the type of the input value. Specifically,
-this value is passed to as the third argument to the PHP :phpfunction:`in_array`
-method when checking to see if a value is in the valid choices array.
+    Setting the strict option of the `Choice` Constraint to `false` has been
+    deprecated as of Symfony 3.2 and the option will be changed to `true` as of 4.0.
```

Comment:
As it's reStructed Text, you have to use double backticks for literals: ``` ``true`` ```

Also, I suggest removing the literal from "Choice constraint". Literals kinda break the read flow, which makes things a little harder to read.